scripts/unpublishFeedGen.ts

```diff
@@ -0,0 +1,39 @@
+import dotenv from 'dotenv'
+import { AtpAgent, BlobRef } from '@atproto/api'
+import fs from 'fs/promises'
+import { ids } from '../src/lexicon/lexicons'
+
+const run = async () => {
+  dotenv.config()
+
+  // YOUR bluesky handle
+  // Ex: user.bsky.social
+  const handle = ''
+
+  // YOUR bluesky password, or preferably an App Password (found in your client settings)
+  // Ex: abcd-1234-efgh-5678
+  const password = ''
```

Comment:
Can we read this from input prompt or .env? Seems safer than people potentially hardcoding password and pushing it

Reply:
Thanks for the review!
Yes, I am aware of the possibility of hard coding and pushing too.
However, I am doing the same once the PR #50 for publishFeedGen has not been merged.
Once PR #50 is merged, we will fix this as well :)

Reply:
I know I'm super late to the party 😭

I added inquirer to get all of these values, since it is just too darn easy to make a commit right now with a credential and, frankly, its a bit easier to just run the script real quick and enter inputs then have to make changes to the file itself. If anyone wants to tweak it still though, feel free.